Homologacao

OscarCarneiro opened the pull request.

rodhideshima committed:
Update Manual_de_Fluxo_do_Tramite.rst

rodhideshima committed:
Update Manual_de_Fluxo_do_Tramite.rst

Solicitações: 
1. Fazer o primeiro fluxo legível;
 
02. ter conversas entre os manuais do tramita;
Exs.: 
- Colocar link do Swagger
- Conceitos da FAQ
 
03. Tramite Cancelado:
 
03a. Tramite Cancelado pelo Remetente
03b. Tramite Cancelado automaticamente

rodhideshima committed:
Update Iniciando_operacoes.rst

Alteração do texto em "Barra de Ferramentas"

rodhideshima committed:
Update Iniciando_operacoes.rst

Inserindo a descrição da funcionalidade "Lixeira" em "Iniciando Operações"

rodhideshima committed:
Update Operacoes_basicas_com_documentos.rst

Inserção de informações sobre a Lixeira (para versões posteriores à versão 5.0) para o caso de exclusão de documentos

rodhideshima committed:
Update Operacoes_basicas_com_documentos.rst

Correção de texto de Excluir documento

rodhideshima committed:
Inserção de imagens Novidades Sei 5.0

Inserção de imagens da Seção 5 (Operações Básicas com Documentos), referentes a novidades do Editor de texto na versão 5.0 do SEI

rodhideshima committed:
Update Operacoes_basicas_com_documentos.rst

Inserção da seção de novidades do Editor de texto para o SEI 5.0

rodhideshima committed:
Inserção de imagens referentes à versão SEI 5.0

Inserção de imagens referentes à versão 5.0 do SEI para a seção de Editor

rodhideshima committed:
Inserção de figuras Editor SEI 5.0

Inserção de figuras referentes às novas funções do Editor SEI 5.0

rodhideshima committed:
Correção de imagem

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update Operacoes_basicas_com_documentos.rst

Correção da posição do texto da seção do Editor novo (versão 5.0); 

Correção de figuras (posicionamento no texto e tamanho)

rodhideshima committed:
Update Operacoes_basicas_com_documentos.rst

Correção de uma nota com referência do tamanho máximo de um upload de arquivo.

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Correção de imagens

Correção de imagens quebradas

pen-docs/docs/source/SEI/Operacoes_basicas_com_documentos > Novas funções da versão 5.0 > Propriedades da tabela

rodhideshima committed:
Update index.rst

Adição de novas classes de órgãos na Introdução

rodhideshima committed:
Update PODER_JUDICIARIO.rst

Alterações no texto

rodhideshima committed:
Update ESTADOS_MUNICIPIOS.rst

Alteração no texto dos órgãos dos Estados e Municípios

rodhideshima committed:
Update CONSELHOS_DE_FISCALIZACAO_PROFISSIONAL.rst

Alteração e correção dos textos de Conselhos de Fiscalização Profissional

rodhideshima committed:
Update MINISTERIOS_PUBLICOS.rst

rodhideshima committed:
Create DEFENSORIAS_PUBLICAS

Inserção do repositório de estrutura das Defensorias Públicas

rodhideshima committed:
Update index.rst

Inserção dos repositórios Defensorias Publicas, Tribunais de Contas e Inativação de Repositório

rodhideshima committed:
Update DEFENSORIAS_PUBLICAS

Inserção do repositório das Defensorias públicas: CONDEGE, DPU, DPDFT, DPE

rodhideshima committed:
Update Operacoes_basicas_com_documentos.rst

Correção de hierarquia das seções

rodhideshima committed:
Create TRIBUNAIS_DE_CONTAS

rodhideshima committed:
Create INATIVACAO_REPOSITORIO

rodhideshima committed:
Rename DEFENSORIAS_PUBLICAS to DEFENSORIAS_PUBLICAS.rst

Correção do tipo de arquivo (acréscimo de .rst)

rodhideshima committed:
Rename INATIVACAO_REPOSITORIO to INATIVACAO_REPOSITORIO.rst

Correção do tipo de arquivo (acréscimo de .rst)

rodhideshima committed:
Rename TRIBUNAIS_DE_CONTAS to TRIBUNAIS_DE_CONTAS.rst

Correção do tipo de arquivo (acréscimo de .rst)

rodhideshima committed:
Inserção de imagens

Inserção de imagens das seções: 
- Tribunais de contas
- Defensorias públicas

rodhideshima committed:
Create Ficha_catalográficaMFT

Ficha catalográfica Manual do Fluxo de Tramite do Tramita.GOV.BR

rodhideshima committed:
Create CreditosMFT

Criação dos Créditos do Manual do Fluxo do Tramite

pedrohpms committed:
Add files via upload

pedrohpms committed:
Rename 1-IO_Tela-Inicial-SSO.gif to 1-IO-Tela_Inicial_SSO.gif

pedrohpms committed:
Update Iniciando_operacoes.rst

pedrohpms committed:
Update Iniciando_operacoes.rst

pedrohpms committed:
Update Iniciando_operacoes.rst

pedrohpms committed:
Update Iniciando_operacoes.rst

pedrohpms committed:
Update Iniciando_operacoes.rst

pedrohpms committed:
Update Iniciando_operacoes.rst

rodhideshima committed:
Update Iniciando_operacoes.rst

Troca de visualização dos itens do Menu Principal de tópicos para tabela

rodhideshima committed:
Update Iniciando_operacoes.rst

Correção da visualização nova do Menu Principal

rodhideshima committed:
Update Iniciando_operacoes.rst

rodhideshima committed:
Update Iniciando_operacoes.rst

rodhideshima committed:
Update Iniciando_operacoes.rst

rodhideshima committed:
Update Iniciando_operacoes.rst

rodhideshima committed:
Update Iniciando_operacoes.rst

Inclusão do antigo "saiba mais" no Menu Principal para dentro da célula

pedrohpms committed:
Update Iniciando_operacoes.rst

pedrohpms committed:
Improve sentence structure in Iniciando_operacoes.rst

pedrohpms committed:
Improve clarity and detail in SEI documentation

Refine descriptions for various functionalities in the SEI system documentation, enhancing clarity and detail.

pedrohpms committed:
Refine descriptions for process control features

pedrohpms committed:
Revise process management feature descriptions

Updated descriptions for process management features to enhance clarity and detail.

pedrohpms committed:
Refine descriptions in Iniciando_operacoes.rst

pedrohpms committed:
Revise process control icon descriptions

Updated descriptions for process control icons and their functionalities.

pedrohpms committed:
Update status descriptions in Iniciando_operacoes.rst

pedrohpms committed:
Align images to center in Iniciando_operacoes.rst

pedrohpms committed:
Improve clarity in process screen documentation

Refine descriptions for process screen and functionalities.

rodhideshima committed:
Add files via upload

Alteração de figura na simbologia dos processos em Manual do SEI > Iniciando Operações > Acesso ao Sistema > Tela de Controle de Processos

pedrohpms committed:
Adjust table widths and image alignment in RST

pedrohpms committed:
Add files via upload

pedrohpms committed:
Fix typo and add description for priority process

pedrohpms committed:
Enhance descriptions for process operations

Expanded descriptions for various process operations, adding details about functionalities and usage.

pedrohpms committed:
Fix typo in 'Adicionar ou Remover marcador' section

pedrohpms committed:
Fix formatting of 'Processo prioritário' section

pedrohpms committed:
Rename docs/source/SEI/_static/images/1-IO-Processo_prioritario.png to docs/source/_static/images/SEI1-IO-Processo_prioritario.png

pedrohpms committed:
Rename SEI1-IO-Processo_prioritario.png to 1-IO-Processo_prioritario.png

pedrohpms committed:
Revise document organization and search descriptions

Updated terminology and clarified features related to document organization and search functionality in the SEI interface.

pedrohpms committed:
Rename 1-IO-Processo_prioritario.png to 1-IO-ProcessoPrioritario.png

pedrohpms committed:
Fix image reference for Processo_prioritario

pedrohpms committed:
Improve clarity and formatting in Iniciando_operacoes.rst

Refactor text for clarity and consistency in the document.

pedrohpms committed:
Fix minor text formatting issues in Iniciando_operacoes.rst

pedrohpms committed:
Revise document operation icon descriptions in SEI

Updated the descriptions of various document operation icons and clarified functionalities in the SEI documentation.

pedrohpms committed:
Enhance document operation descriptions in SEI

Added descriptions for document operations and clarified document types.

pedrohpms committed:
Improve text formatting and clarity in Iniciando_operacoes.rst

Refine text formatting and improve clarity in various sections.

pedrohpms committed:
Remove 'Processo_prioritario' image reference

Removed image reference for 'Processo_prioritario' from the documentation.

pedrohpms committed:
Enhance 'Incluir documento' description in SEI

Updated the description of the 'Incluir documento' feature to include details about generating documents from templates and standard text.

rodhideshima committed:
Create index.rst

Introdução ao MTO do Protocolo GOV.BR para Sydle

rodhideshima committed:
Update index.rst

Inserção do conteúdo inicial da introdução do MTO para Sydle

rodhideshima committed:
Create INTRODUCAO

Criação do documento de introdução e inserção do seu conteúdo

rodhideshima committed:
Create DEFINICOES

Criação do documento DEFINICOES e inserção do conteúdo

rodhideshima committed:
Create REQUISITOS_BASICOS.rst

Criação do documento REQUISITOS_BASICOS e inserção do seu conteúdo

rodhideshima committed:
Rename INTRODUCAO to INTRODUCAO.rst

correção do tipo de arquivo

rodhideshima committed:
Rename DEFINICOES to DEFINICOES.rst

rodhideshima committed:
Create SOLICITACAO_ADESAO.rst

Criação do documento SOLICITACAO_ADESAO e inserção do conteúdo

rodhideshima committed:
Create CONFIGURACAO_SPE.rst

Criação do documento CONFIGURACAO_SPE e inserção inicial do conteúdo

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

rodhideshima committed:
Update index.rst

rodhideshima committed:
Create texto

rodhideshima committed:
Inserção de primeiros arquivos gifs

Inserção de primeiros arquivos gifs

rodhideshima committed:
Update index.rst

rodhideshima committed:
Create CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

Criação do documento CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE e inserção do conteúdo inicial

rodhideshima committed:
Create ATENDIMENTO_DA_SOLICITACAO_NA_PLATAFORMA_SYDLE.rst

Criação do documento ATENDIMENTO_DA_SOLICITACAO_NA_PLATAFORMA_SYDLE e inserção do conteúdo

rodhideshima committed:
Create TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

Criação do documento TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO e inserção inicial do conteúdo

rodhideshima committed:
Create CONFERINDO_O_PROCESSO_NO_SPE.rst

Criação do documento CONFERINDO_O_PROCESSO_NO_SPE e inserindo seu conteúdo inicial

rodhideshima committed:
Create UTILIZANDO_MODULO_RESPOSTA.rst

Criação do documento UTILIZANDO_MODULO_RESPOSTA e inserção do seu conteúdo inicial

rodhideshima committed:
Create GERENCIAMENTO_DE_USUARIOS.rst

Criação do documento GERENCIAMENTO_DE_USUARIOS e inserção do seu conteúdo inicial

rodhideshima committed:
Create GLOSSARIO_DE_ICONES.rst

Criação do documento GLOSSARIO_DE_ICONES e inserção do seu conteúdo inicial

rodhideshima committed:
Create ABRIR_CHAMADO.rst

Criação do documento ABRIR_CHAMADO e inserção do seu conteúdo inicial

rodhideshima committed:
Create OUTRAS_INFORMACOES.rst

Criação do documento OUTRAS_INFORMACOES e inserção do seu conteúdo inicial

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update INTRODUCAO.rst

rodhideshima committed:
Update DEFINICOES.rst

rodhideshima committed:
Update REQUISITOS_BASICOS.rst

rodhideshima committed:
Update SOLICITACAO_ADESAO.rst

rodhideshima committed:
Atualização do pacote de gifs

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

Inserção do caminho dos arquivos gifs

rodhideshima committed:
Update ACESSO_AO_AMBIENTE_DE_PRODUCAO.rst

Alteração solicitada pela DIVIN

rodhideshima committed:
Update DEFINICOES.rst

Melhora na visualização das definições do Manual técnico Operacional para Sydle

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update SOLICITACAO_ADESAO.rst

Inserção do link mencionando quais anexos são necessários para a solicitação de Adesão para a implantação

rodhideshima committed:
Create CONFIGURACAO_DA_INTEGRACAO_DO_PROTOCOLOGOVBR_COM_SPE.rst

Inserção da sessão "Configuração da Integração do PROTOCOLO.GOV.BR com o SPE"

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

Correção:
retirada do título erroneamente inserido "Manual Técnico Operacional do Protocolo GOV.BR para Sydle" para essa seção.

rodhideshima committed:
Create CONFIGURACAO_INTEGRACAO.rst

Inserção de ícone "Mais opções"no texto

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update GLOSSARIO_DE_ICONES.rst

Inserção dos ícones para o Glossário

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update ABRIR_CHAMADO.rst

rodhideshima committed:
Update DEFINICOES.rst

rodhideshima committed:
Update INTRODUCAO.rst

Inserção do hiperlink 
https://www.in.gov.br/en/web/dou/-/portaria-seges/me-n-10.988-de-23-de-dezembro-de-2022-453752290, referente à Portaria SEGES/ME Nº 10.988, de 23 de dezembro de 2022

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update CONFIGURACAO_INTEGRACAO.rst

rodhideshima committed:
Update CONFIGURACAO_DA_INTEGRACAO_DO_PROTOCOLOGOVBR_COM_SPE.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update SOLICITACAO_DE_ACESSO.rst

Alteração solicitada pelo Vinicius na substituição do tipo de solicitante do Art. 4º, III - 

MTO > Solicitação de Acesso > Preenchimento do Cadastro

rodhideshima committed:
Add files via upload

Gifs

rodhideshima committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

rodhideshima committed:
Update CONFERINDO_O_PROCESSO_NO_SPE.rst

rodhideshima committed:
Update CONFERINDO_O_PROCESSO_NO_SPE.rst

rodhideshima committed:
Update index.rst

Correções pontuais: 

- Eliminação do texto "no âmbito da administração pública federal direta, autárquica e fundacional ";
- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI) 

rodhideshima committed:
Update INTRODUCAO.rst

Correções pontuais: 

- Padronização do nome Protocolo GOV.BR

rodhideshima committed:
Update DEFINICOES.rst

Correções pontuais: 

- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI). No caso, ficou SEGES/MGI, por conta da citação da Secretaria de Gestão

rodhideshima committed:
Update REQUISITOS_BASICOS.rst

Correções pontuais: 

- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI)

rodhideshima committed:
Update SOLICITACAO_ADESAO.rst

Correções pontuais: 

- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI)

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

Correções pontuais: 

- Eliminação do texto "no âmbito da administração pública federal direta, autárquica e fundacional ";
- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI)

rodhideshima committed:
Update CONFIGURACAO_DA_INTEGRACAO_DO_PROTOCOLOGOVBR_COM_SPE.rst

Correções pontuais: 

- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI)

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

Correções pontuais: 

- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI)

rodhideshima committed:
Update ATENDIMENTO_DA_SOLICITACAO_NA_PLATAFORMA_SYDLE.rst

Correções pontuais: 


- Padronização do nome Protocolo GOV.BR
- Padronização do nome + Sigla Ministério da Gestão e da Inovação em Serviços Públicos (MGI)

rodhideshima committed:
Update ABRIR_CHAMADO.rst

Correções pontuais: 


- Padronização do nome Protocolo GOV.BR

rodhideshima committed:
Update OUTRAS_INFORMACOES.rst

Correções pontuais: 


- Padronização do nome Protocolo GOV.BR

rodhideshima committed:
Update SOLICITACAO_ADESAO.rst

Alteração de texto, para atender o procedimento de Adesão

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

Alterações pontuais: 

- Padronização do título
- Exclusão do item 11

rodhideshima committed:
Update CONFIGURACAO_SPE.rst

rodhideshima committed:
Update CONFIGURACAO_DA_INTEGRACAO_DO_PROTOCOLOGOVBR_COM_SPE.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Rename 01_Parametros_para_integracao_SPE_doc-adicional.gif.gif to 01_Parametros_para_integracao_SPE_doc-adicional.gif

rodhideshima committed:
Rename 01_Parametros_para_integracao_SPE_doc-opcional.gif.gif to 01_Parametros_para_integracao_SPE_doc-opcional.gif

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update ATENDIMENTO_DA_SOLICITACAO_NA_PLATAFORMA_SYDLE.rst

Alterações solicitadas: 

- Troca do texto do hiperlink da Sydle

- Padronização da caixa de Atenção

- Correção do texto (última linha da caixa de atenção)

rodhideshima committed:
Add files via upload

Inclusão do Icone_Filtro

rodhideshima committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update CONFERINDO_O_PROCESSO_NO_SPE.rst

rodhideshima committed:
Create MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update index.rst

Ocultação do arquivo antigo do módulo de assinatura avançada e apresentação do novo módulo de Assinatura avançada (arquivo SEIAssina)

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

Correções e alterações solicitadas

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Update MODULO_SEIAssina.rst

rodhideshima committed:
Add files via upload

Ícones da sessão de Gerenciamento de usuários

rodhideshima committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

rodhideshima committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

rodhideshima committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

Imagens do arquivo de Módulo resposta

rodhideshima committed:
Update UTILIZANDO_MODULO_RESPOSTA.rst

rodhideshima committed:
Update CONFIGURACAO_INTEGRACAO.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update CONFERINDO_O_PROCESSO_NO_SPE.rst

rodhideshima committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

rodhideshima committed:
Update CONFERINDO_O_PROCESSO_NO_SPE.rst

rodhideshima committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

rodhideshima committed:
Update ABRIR_CHAMADO.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update OUTRAS_INFORMACOES.rst

rodhideshima committed:
Update UTILIZANDO_MODULO_RESPOSTA.rst

rodhideshima committed:
Update OUTRAS_INFORMACOES.rst

rodhideshima committed:
Update CONFERINDO_O_PROCESSO_NO_SPE.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update CONFIGURACAO_DA_INTEGRACAO_DO_PROTOCOLOGOVBR_COM_SPE.rst

rodhideshima committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

rodhideshima committed:
Update UTILIZANDO_MODULO_RESPOSTA.rst

rodhideshima committed:
Add files via upload

rodhideshima committed:
Update MODULO_SEIAssina.rst

Adição das sessões Ficha catalográfica e Créditos

rodhideshima committed:
Update AssinAvan_01-Solicitacao_Login_GOVBR.png

rodhideshima committed:
Update AssinAvan_01-Solicitacao_Dados-Org-Solicit.png

Alteração da figura

rodhideshima committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

rodhideshima committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

rodhideshima committed:
Upload de imagens atualizadas

rodhideshima committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

rodhideshima committed:
Merge branch 'homologacao' of https://github.com/pengovbr/pen-docs into homologacao

rodhideshima committed:
Add files via upload

rodhideshima committed:
Add files via upload

caduvieira commented:
Sugiro fechar esse PR visto que vocês não estão fazendo o merge em nenhum momento. @rodhideshima @pedrohpms @marcoabraga

rodhideshima committed:
Add files via upload

Upload de figura do escudo do MGI para a sessão Créditos

rodhideshima committed:
Update MODULO_SEIAssina.rst

Inclusão do cabeçalho da sessão de créditos

rodhideshima committed:
Upload de Guia_Op_recusa

Upload do Guia Operacional para recusa de acesso à Plataforma Tramita GOV.BR v1.0

rodhideshima committed:
Update index.rst

Inserção do capítulo do Guia de Recusa de Acesso da plataforma Tramita GOV.BR

rodhideshima committed:
Update Index.rst

rodhideshima committed:
Merge branch 'homologacao' of https://github.com/pengovbr/pen-docs into homologacao

rodhideshima committed:
Upload da estrutura do NIpe

Upload da estrutura do NIpe

rodhideshima committed:
Create NIPE_SEI.png

rodhideshima committed:
Update index.rst

rodhideshima committed:
Update index.rst

rodhideshima committed:
Create O_que_e_o_NIPE.rst

rodhideshima committed:
Create Estrutura_do_Numero.rst

rodhideshima committed:
Update index.rst

rodhideshima committed:
Create Configuracao_no_SEI.rst

rodhideshima committed:
Rename Index.rst to index.rst

rodhideshima committed:
Update index.rst

rodhideshima committed:
Update index.rst

rodhideshima committed:
Update index.rst

rodhideshima committed:
Create Ficha_Catalografica.rst

rodhideshima committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

Solicitações do Pedro: 

- Correção do nome do módulo. O correto é Módulo Assinatura Eletrônica (sem preposição);
- Remoção do texto que antecede cada imagem;
- Melhora do texto sobre pré-requisito do Login GOV.BR

rodhideshima committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

Correções solicitadas

rodhideshima committed:
Upload de arquivos do NIPE

rodhideshima committed:
Merge branch 'homologacao' of https://github.com/pengovbr/pen-docs into homologacao

rodhideshima committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

rodhideshima committed:
Atualização de imagens

Atualização de imagens sem indicativo de seta

marcoabraga committed:
Add files via upload

Atualização do banner principal do portal

rodhideshima committed:
Update Gestao_documental.rst

Solicitação da Duane 15/05

rodhideshima committed:
Alteração de pasta

Alteração de pasta do conteúdo.

rodhideshima committed:
Upload de arquivos do NIPE para o SEI

Upload de arquivos do NIPE para o SEI

Carol-Freirer committed:
Update Gestao_documental.rst

alteração para contemplar novo decreto de eliminação

Carol-Freirer committed:
Update Gestao_documental.rst

ajuste texto

Carol-Freirer committed:
Update Gestao_documental.rst

ajuste de texto

Carol-Freirer committed:
Update Gestao_documental.rst

ajuste de texto

rodhideshima committed:
Update index.rst

Inserção dos links dos manuais do NIPE

camilacorgosinho committed:
Create Manual_de_Fluxo_do_Tramite.rst

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/Manual_de_Fluxo_do_Tramite.rst to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/Manual_de_Fluxo_do_Tramite01.rst

camilacorgosinho committed:
Delete docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/Manual_de_Fluxo_do_Tramite.rst

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/Manual_de_Fluxo_do_Tramite01.rst to docs/source/TRAMITA.GOV.BR/Manual_de_Fluxo_do_Tramite.rst

camilacorgosinho committed:
Rename Manual_de_Fluxo_do_Tramite01.rst to Manual_de_Fluxo_do_Tramite.rst

camilacorgosinho committed:
Update index.rst

Incluido Manual Fluxo Tramite

camilacorgosinho committed:
Create MANUAL_FLUXO_TRAMITE

camilacorgosinho committed:
Delete docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/Manual_de_Fluxo_do_Tramite.rst to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/Manual_de_Fluxo_do_Tramite.rst

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/CreditosMFT to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/CreditosMFT

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/Ficha_catalográficaMFT to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/Ficha_catalográficaMFT

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_01-Tramite_Realizado.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_01-Tramite_Realizado.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/Padrao_de_Criacao_de_Repositorio-V1.0.pdf to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/Padrao_de_Criacao_de_Repositorio-V1.0.pdf

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/Padrao_de_Criacao_de_Repositorio-V2.0.pdf to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/Padrao_de_Criacao_de_Repositorio-V2.0.pdf

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_02-Tramite_Recusado_pelo_destinatario.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_02-Tramite_Recusado_pelo_destinatario.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_03-Tramite_Cancelado_pelo_remetente.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_03-Tramite_Cancelado_pelo_remetente.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Atualiz_01-SHA1_Fingerprint.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Atualiz_01-SHA1_Fingerprint.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Atualiz_02-Certif_cript.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Atualiz_02-Certif_cript.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_01-envio_proc_adm_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_01-envio_proc_adm_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_02-envio_DocAv_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_02-envio_DocAv_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_03-envio_CompDig_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_03-envio_CompDig_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_04-Recibo_conclusao_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_04-Recibo_conclusao_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_05-Receb_Metadados_CompDig_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_05-Receb_Metadados_CompDig_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_06-AssEnv_RecConc_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_06-AssEnv_RecConc_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_07-Receber_RecConc_v02.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_07-Receber_RecConc_v02.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_08-RecusTram.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_08-RecusTram.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_09-FinalizRecusTram.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_09-FinalizRecusTram.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_10-CancTram.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_10-CancTram.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/Fluxo_tramite_Cenario_11-CancTramAutom.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/Fluxo_tramite_Cenario_11-CancTramAutom.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/ksnip_20250611-193302.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/ksnip_20250611-193302.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/_static/images/título-Manual-do-Tramita.png to docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/título-Manual-do-Tramita.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/MANUAL_FLUXO_TRAMITE/_static/images/título-Manual-do-Tramita.png to docs/source/TRAMITA.GOV.BR/_static/images/título-Manual-do-Tramita.png

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Create index

camilacorgosinho committed:
Rename index to index.rst

camilacorgosinho committed:
Rename CreditosMFT to CreditosMFT.rst

camilacorgosinho committed:
Rename Ficha_catalográficaMFT to Ficha_catalográficaMFT.rst

camilacorgosinho committed:
Rename Ficha_catalográficaMFT.rst to Ficha_catalograficaMFT.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename index.rst.backup to index backup.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename index.rst to index.rst.teste

camilacorgosinho committed:
Rename index.rst.teste to index.rst

camilacorgosinho committed:
Rename index backup.rst to index backup.rstt.este

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update CreditosMFT.rst

caduvieira committed:
cicd: atualiza os para a mais recente

camilacorgosinho committed:
Update CreditosMFT.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Create O_que_e_o_NIPE_V2.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Update O_que_e_o_NIPE_V2.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update Estrutura_do_Numero.rst

camilacorgosinho committed:
Rename O_que_e_o_NIPE_V2.rst to 01O_que_e_o_NIPE_V2.rst

camilacorgosinho committed:
Rename Estrutura_do_Numero.rst to 02Estrutura_do_Numero.rst

camilacorgosinho committed:
Rename Configuracao_no_SEI.rst to 03Configuracao_no_SEI.rst

camilacorgosinho committed:
Rename Gestao_de_Unidades_e_Sequenciais.rst to 04Gestao_de_Unidades_e_Sequenciais.rst

camilacorgosinho committed:
Rename Exemplo_Pratico_de_Preenchimento.rst to 05Exemplo_Pratico_de_Preenchimento.rst

camilacorgosinho committed:
Rename Observacoes_Importantes.rst to 06Observacoes_Importantes.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Update 03Configuracao_no_SEI.rst

camilacorgosinho committed:
Update 02Estrutura_do_Numero.rst

camilacorgosinho committed:
Update 03Configuracao_no_SEI.rst

camilacorgosinho committed:
Update 04Gestao_de_Unidades_e_Sequenciais.rst

camilacorgosinho committed:
Update 05Exemplo_Pratico_de_Preenchimento.rst

camilacorgosinho committed:
Update 06Observacoes_Importantes.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update Login_externo_GovBR.rst

camilacorgosinho committed:
Update Login_externo_GovBR.rst

camilacorgosinho committed:
Create Login_externo_GovBR.rst.backupdefeso

camilacorgosinho committed:
Update Login_externo_GovBR.rst.backupdefeso

camilacorgosinho committed:
Rename Login_externo_GovBR.rst.backupdefeso to Login_externo_GovBR.backupdefeso

camilacorgosinho committed:
Create Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.backupdefeso2026

camilacorgosinho committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.backupdefeso2026

camilacorgosinho committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

camilacorgosinho committed:
Create ADMINISTRACAO.backupdefeso2026

camilacorgosinho committed:
Update ADMINISTRACAO.backupdefeso2026

camilacorgosinho committed:
Update Login_externo_GovBR.rst

camilacorgosinho committed:
Update Assinatura_Avancada_Processo_de_solicitacao_e_habilitacao.rst

camilacorgosinho committed:
Update ADMINISTRACAO.rst

camilacorgosinho committed:
Create SOLICITACAO_DE_ACESSO.backupdefeso2026

camilacorgosinho committed:
Update SOLICITACAO_DE_ACESSO.backupdefeso2026

camilacorgosinho committed:
Update SOLICITACAO_DE_ACESSO.rst

camilacorgosinho committed:
Create images

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Rename images to teste

camilacorgosinho committed:
Rename Banner_brasao_antigo_DTGES.png to Gestao_do_portalBanner_brasao_antigo_DTGES.png

camilacorgosinho committed:
Rename docs/source/TRAMITA.GOV.BR/MANUAIS_PERFIS/iimages/Gestao_do_portalBanner_brasao_antigo_DTGES.png to docs/source/TRAMITA.GOV.BR/MANUAIS_PERFIS/Gestao_do_portal/images/Banner_brasao_antigo_DTGES.png

camilacorgosinho committed:
Update CREDITOS.rst

camilacorgosinho committed:
Delete docs/source/TRAMITA.GOV.BR/MANUAIS_PERFIS/Gestao_do_portal/teste

camilacorgosinho committed:
Rename MODULO_SEIAssina.rst to MODULO_SEIAssina.backupdefeso2026

camilacorgosinho committed:
Create MODULO_SEIAssina.rst

camilacorgosinho committed:
Update MODULO_SEIAssina.rst

camilacorgosinho committed:
Rename index.rst to index.backupdefeso2026

camilacorgosinho committed:
Create index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename index.rst to index.backupdefeso2026

camilacorgosinho committed:
Create index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename PROTOCOLO.rst to PROTOCOLO.backupdefeso2026

camilacorgosinho committed:
Create PROTOCOLO.rst

camilacorgosinho committed:
Update PROTOCOLO.rst

camilacorgosinho committed:
Rename index.rst to index.backupdefeso2026

camilacorgosinho committed:
Create index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename index.rst to index.backupdefeso2026

camilacorgosinho committed:
Create index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename index.rst to index.backupdefeso2026

camilacorgosinho committed:
Create index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename ABRIR_CHAMADO.rst to ABRIR_CHAMADO.backupdefeso2026

camilacorgosinho committed:
Create ABRIR_CHAMADO.rst

camilacorgosinho committed:
Update ABRIR_CHAMADO.rst

camilacorgosinho committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

camilacorgosinho committed:
Rename Produtividade por analista.gif to Produtividade_por_analista.gif

camilacorgosinho committed:
Rename Recorrência de solicitações por tipo.gif to Recorrência_de_solicitações_por_tipo.gif

camilacorgosinho committed:
Rename Solicitações devolvidas para ajustes por analista.gif to Solicitações_devolvidas_para_ajustes_por_analista.gif

camilacorgosinho committed:
Rename Solicitações triadas por analista.gif to Solicitações_triadas_por_analista.gif

camilacorgosinho committed:
Rename Tempo médio para realização da triagem.gif to Tempo_médio_para_realização_da_triagem.gif

camilacorgosinho committed:
Update GERENCIAMENTO_DE_USUARIOS.rst

camilacorgosinho committed:
Rename GERENCIAMENTO_DE_USUARIOS.rst to FUNCIONALIDADES_DE_GESTAO.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Rename Recorrência_de_solicitações_por_tipo.gif to Recorrencia_de_solicitacoes_por_tipo.gif

camilacorgosinho committed:
Rename Solicitações_devolvidas_para_ajustes_por_analista.gif to Solicitacoes_devolvidas_para_ajustes_por_analista.gif

camilacorgosinho committed:
Rename Solicitações_triadas_por_analista.gif to Solicitacoes_triadas_por_analista.gif

camilacorgosinho committed:
Rename Tempo_médio_para_realização_da_triagem.gif to Tempo_medio_para_realizacao_da_triagem.gif

camilacorgosinho committed:
Update FUNCIONALIDADES_DE_GESTAO.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Update FUNCIONALIDADES_DE_GESTAO.rst

camilacorgosinho committed:
Update FUNCIONALIDADES_DE_GESTAO.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Rename index.rst to index_backupdefeso2026.rst

camilacorgosinho committed:
Create index.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Update 03Configuracao_no_SEI.rst

camilacorgosinho committed:
Update index.rst

camilacorgosinho committed:
Create index.backupdefeso2026

camilacorgosinho committed:
Update CONFIGURACAO_DE_UMA_SOLICITACAO_NA_SYDLE.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst

camilacorgosinho committed:
Add files via upload

camilacorgosinho committed:
Update TRIAGEM_PELA_EQUIPE_DE_PROTOCOLO.rst
